src/BloomExe/CollectionTab/LibraryListView.cs

```diff
@@ -92,19 +91,48 @@ enum ButtonManagementStage
 			_showHistoryMenu.Visible = _showNotesMenu.Visible = Settings.Default.ShowSendReceive;
 
 			if(Settings.Default.ShowExperimentalCommands)
-				_settingsProtectionHelper.ManageComponent(_exportToXMLForInDesignToolStripMenuItem);//we are restriting it because it opens a folder from which the user could do damage
+				_settingsProtectionHelper.ManageComponent(_exportToXMLForInDesignToolStripMenuItem);//we are restricting it because it opens a folder from which the user could do damage
 			_exportToXMLForInDesignToolStripMenuItem.Visible = Settings.Default.ShowExperimentalCommands;
 
 			SetupBookDropdownIcon();
 			_bookContextMenu.Closed += _bookContextMenu_Closed;
+			_bookContextMenu.Opening += _bookContextMenu_Opening;
+		}
+
+		// adjust the context menu item visibility based on what sort of collection we're in
+		void _bookContextMenu_Opening(object sender, CancelEventArgs e)
+		{
+			var btn = (sender as ContextMenuStrip).SourceControl as Button;
```

Comment:
Where are the menu items created? Seems it would be useful to have a cross-reference comment there saying that currently these menu items are all disabled for factory books, and most for bloom library...if someone adds a new one they should be prompted to think about which collections should have it.